packages/client/src/pages/settings/index.vue

```diff
@@ -212,6 +213,8 @@ const component = computed(() => {
 	return null;
 });
 
+promptLoginOrRedirect();
```

Comment:
I think this would usually be a case for `pleaseLogin();`. Maybe it should be used here too. Although in my opinion, the user experience of `pleaseLogin` is not good since there is no way to log in when you see the dialog, so maybe it should be refactored.

But having two scripts that do the same thing would be against DRY.